packages/vendure-plugin-stripe-subscription/src/api/stripe-subscription.controller.ts

```diff
@@ -115,14 +119,22 @@ export class ShopResolver {
 }
 
 @Resolver('OrderLine')
-export class ShopOrderLinePricingResolver {
-  constructor(private subscriptionService: StripeSubscriptionService) {}
+export class OrderLinePricingResolver {
+  constructor(
+    private entityHydrator: EntityHydrator,
+    private subscriptionService: StripeSubscriptionService
+  ) {}
 
   @ResolveField()
   async subscriptionPricing(
     @Ctx() ctx: RequestContext,
     @Parent() orderLine: OrderLineWithSubscriptionFields
   ): Promise<StripeSubscriptionPricing | undefined> {
+    if (ctx.apiType !== 'shop') {
+      await this.entityHydrator.hydrate(ctx, orderLine, {
+        relations: ['productVariant'],
+      });
+    }
```

Comment:
Ahh, this relation is not available in the admin-api? You can still do the hydration without if statement, because the EntityHydrator will check if the relation is already available when you call `hydrate`
```suggestion
    await this.entityHydrator.hydrate(ctx, orderLine, {
      relations: ['productVariant'],
    });
```